android/samples/test-app/src/main/res/layout/ddm_form.xml

```diff
@@ -1,22 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     >
 
-    <com.liferay.mobile.screens.thingscreenlet.screens.ThingScreenlet
+    <com.liferay.mobile.screens.ddm.form.DDMFormScreenlet
         android:id="@+id/ddm_form_screenlet"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        app:formInstanceId="37105"
```

Comment:
this id should be in server_context file

Reply:
What about an app with more than one Form?

Reply:
you mean, only for test app ?

Reply:
I mean for test-app :) All our ids are in server_context file, so if you want to try one screenlet, you go there and edit the id